woocommerce/woocommerce
# Revert #28187

### All Submissions:


### Changes proposed in this Pull Request:

This PR reverts the theme support refactor in #28187.

The default functionality within the `ThemeSupport` class has an unintended side-effect of breaking the existing default functionality within the `wc_get_theme_support()` function. Since the default set in the support class is prioritized over the one given to `ThemeSupport::get_option()`, the WordPress options set in the Customizer are never used for images.

Closes #29950.

### How to test the changes in this Pull Request:

1. Set the Twenty-TwentyOne Theme.
2. Go to Customize > WooCommerce > Product Images and verify that the sliders are not present anymore.
3. Set the Storefront Theme and make sure that the sliders are also not present, and that the images look sized correctly.

### Other information:


### Changelog entry

**Reverts an unreleased PR**
